### PR TITLE
Remove SkipIfCurrentRuntimeIsCoreClrAttribute

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Found this bug while working on https://github.com/aspnet/IISIntegration/pull/345. Updating to use RuntimeInformation.FrameworkDescription instead

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsCoreClrAttribute.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public bool IsMet
         {
             get
             {
-                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("coreclr");
+                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("dotnet");
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsCoreClrAttribute.cs]: You could use `RuntimeInformation.FrameworkDescription`. I don't think looking for the process name is the correct check (for instance standalone apps don't have this name).

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsClrAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    /// Skips a test if the runtime used to run the test is Clr.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class SkipIfCurrentRuntimeIsClrAttribute : Attribute, ITestCondition
+    {
+        public bool IsMet
+        {
+            get
+            {
+                return !Process.GetCurrentProcess().ProcessName.ToLower().Contains("testhost");
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsClrAttribute.cs]: Where's this used?

Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsClrAttribute.cs]: I was going to use it in IISIntegration functional tests. Also I don't see why we only have the attribute for skipping CoreCLR but not CLR.

File: src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsCoreClrAttribute.cs (hunk 1 above)
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Server.IntegrationTesting/xunit/SkipIfCurrentRuntimeIsCoreClrAttribute.cs]: Ah that would be better. I was wondering why this looks at the process name but I guess it's just really old code.

**Comment by JunTaoLuo** [comment]
Apparently there is `FrameworkSkipCondition(RuntimeFrameworks.CLR)` so these attributes are redundant. Will remove

File: Hosting.sln [hunk 3]
```diff
@@ -1,35 +1,34 @@
-
```
Comment (davidfowl) [inline Hosting.sln]: What happened here? can you revert this?

Comment (JunTaoLuo) [inline Hosting.sln]: VS did something... will revert.

**Pull request merged**